operator/test/test.sh

```diff
@@ -182,14 +182,14 @@ test_results() {
   test_pipelines
   echo -n "  - Results in database: "
 
-  # Prepare a custom Service Account that will be used for debugging purposes
-  if ! kubectl get serviceaccount tekton-results-debug -n tekton-results >/dev/null 2>&1; then
-    kubectl create serviceaccount tekton-results-debug -n tekton-results
+  # Service Account to test tekton-results
+  if ! kubectl get serviceaccount tekton-results-tests -n "$NAMESPACE" >/dev/null 2>&1; then
```

Comment:
The account name is used a lot. Maybe use a variable?

Reply:
Done